tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -380,7 +385,7 @@ def _mul_handler(self, trans, node):
                 multiplier_input_id = i
                 multiplier_input_node = input_node
 
-        if not multiplier_input_node.is_const():
+        if multiplier_input_node is None or not multiplier_input_node.is_const():

```

Comment:
thanks for fixing that. add a comment for better understanding?

Reply:
done